.github/workflows/ci-backend-cql.yml

```diff
@@ -72,6 +72,8 @@ jobs:
             args: "-Pcassandra3-murmur-ssl -Dtest=\"**/diskstorage/cql/CQLStoreTest.java\""
           - module: cql
             args: "-Pcassandra3-murmur-client-auth -Dtest=\"**/diskstorage/cql/CQLStoreTest.java\""
+          - module: cql
```

Comment:
This wouldn't correctly if some is using the [cql-tests] flags. Could you clone this section to just test scylla?

Reply:
I'm not sure I'm following exactly ? can you elaborate a bit more ?

Reply:
This issue helps if the if in following lines triggers: https://github.com/JanusGraph/janusgraph/blob/master/.github/workflows/ci-backend-cql.yml#L90-L92

Here an idea what i would do and small improvement
```patch
diff --git a/.github/workflows/ci-backend-cql.yml b/.github/workflows/ci-backend-cql.yml
index 9a549c7d9..d4bd9c3da 100644
--- a/.github/workflows/ci-backend-cql.yml
+++ b/.github/workflows/ci-backend-cql.yml
@@ -72,6 +72,8 @@ jobs:
             args: "-Pcassandra3-murmur-ssl -Dtest=\"**/diskstorage/cql/CQLStoreTest.java\""
           - module: cql
             args: "-Pcassandra3-murmur-client-auth -Dtest=\"**/diskstorage/cql/CQLStoreTest.java\""
+          - module: cql
+            args: "-Pscylladb -Dcassandra.docker.image=scylladb/scylla -Dcassandra.docker.version=4.3.0"
     steps:
       - uses: actions/checkout@v2
       - uses: actions/cache@v2
@@ -85,7 +87,41 @@ jobs:
           java-version: 1.8
       - run: mvn clean install --projects janusgraph-${{ matrix.module }} ${{ env.BUILD_MAVEN_OPTS }} ${{ matrix.install-args }}
       - run: mvn verify --projects janusgraph-${{ matrix.module }} ${{ env.VERIFY_MAVEN_OPTS }} ${{ matrix.args }}
-      - name: cql 3.0.18
-        if: "github.event_name == 'push' && contains(github.event.head_commit.message, '[cql-tests]') || github.event_name == 'pull_request' && contains(github.event.pull_request.title, '[cql-tests]')"
-        run: mvn verify --projects janusgraph-${{ matrix.module }} -Dcassandra.docker.version=3.0.18' ${{ env.VERIFY_MAVEN_OPTS }} ${{ matrix.args }}
+      - uses: codecov/codecov-action@v1
+
+  full-tests:
+    runs-on: ubuntu-20.04
+    if: "github.event_name == 'push' && contains(github.event.head_commit.message, '[cql-tests]') || github.event_name == 'pull_request' && contains(github.event.pull_request.title, '[cql-tests]')"
+    needs: build-all
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - module: cql
+            args: "-Pcassandra3-byteordered -Dtest=\"**/diskstorage/cql/*\""
+          - module: cql
+            args: "-Pcassandra3-murmur -Dtest=\"**/diskstorage/cql/*\""
+          - module: cql
+            args: "-Pcassandra3-byteordered -Dtest=\"**/graphdb/cql/*\""
+          - module: cql
+            args: "-Pcassandra3-murmur -Dtest=\"**/graphdb/cql/*\""
+          - module: cql
+            args: "-Pcassandra3-murmur -Dtest=\"**/hadoop/*\""
+          - module: cql
+            args: "-Pcassandra3-murmur-ssl -Dtest=\"**/diskstorage/cql/CQLStoreTest.java\""
+          - module: cql
+            args: "-Pcassandra3-murmur-client-auth -Dtest=\"**/diskstorage/cql/CQLStoreTest.java\""
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/cache@v2
+        with:
+          path: ~/.m2/repository
+          key: ${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}
+          restore-keys: |
+            ${{ runner.os }}-maven-
+      - uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+      - run: mvn clean install --projects janusgraph-${{ matrix.module }} ${{ env.BUILD_MAVEN_OPTS }} ${{ matrix.install-args }}
+      - run: mvn verify --projects janusgraph-${{ matrix.module }} -Dcassandra.docker.version=3.0.18' ${{ env.VERIFY_MAVEN_OPTS }} ${{ matrix.args }}
       - uses: codecov/codecov-action@v1
diff --git a/janusgraph-cql/pom.xml b/janusgraph-cql/pom.xml
index badee09ca..061607a0a 100644
--- a/janusgraph-cql/pom.xml
+++ b/janusgraph-cql/pom.xml
@@ -386,6 +386,7 @@
                 <test.skip.murmur>true</test.skip.murmur>
                 <test.skip.murmur-serial>true</test.skip.murmur-serial>
                 <test.skip.murmur-ssl>true</test.skip.murmur-ssl>
+                <test.cql.excludes>${test.excluded.groups}</test.cql.excludes>
             </properties>
         </profile>
     </profiles>
```

Reply:
@farodin91 took me a while to how this change is related, I think I've got it now...

Reply:
@farodin91 now I've got what you meant, add this to the PR